contracts/tg4-stake/src/msg.rs

```diff
@@ -36,11 +36,14 @@ pub struct InstantiateMsg {
 #[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
 #[serde(rename_all = "snake_case")]
 pub enum ExecuteMsg {
-    /// Bond will bond all staking tokens sent with the message and update membership points
-    Bond {},
+    /// Bond will bond all staking tokens sent with the message and update membership points.
+    /// The optional `vesting_tokens` will be staked (delegated) as well, if set.
+    Bond { vesting_tokens: Option<Coin> },
     /// Unbond will start the unbonding process for the given number of tokens.
     /// The sender immediately loses points from these tokens, and can claim them
-    /// back to his wallet after `unbonding_period`
+    /// back to his wallet after `unbonding_period`.
+    /// Tokens will be unbonded from the liquid pool first, and then from the vesting pool
```

Comment:
Nitpicking: `liquid pool` or `liquidity pool`? Just for consistency

Reply:
Liquid I think it's OK. "Liquidity pool" is a more general term. In fact, just "liquid stake" would be better.